src/main/java/gov/nasa/pds/api/engineering/elasticsearch/ElasticSearchUtil.java

```diff
@@ -24,6 +25,13 @@ static public String jsonPropertyToElasticProperty(String jsonProperty) {
 		
 	}
 	
+	static private void addReference (ArrayList<Reference> to, String ID, String baseURL)
+	{
+		Reference reference = new Reference();
+		reference.setHref(baseURL + "/products/" + reference.getId());
```

Comment:
did you remove the ID assignation ? We still need it, we just don't want the .orex suffix to be added.

Reply:
done